sched/misc/assert.c

```diff
@@ -674,6 +674,14 @@ void _assert(FAR const char *filename, int linenum,
 #else
       for (; ; )
         {
+#ifdef CONFIG_ARCH_LEDS
+          /* FLASH LEDs a 2Hz */
+          // board_autoled_on(LED_PANIC);
+          printf("ON\n");
+          up_mdelay(250);
+          printf("OFF\n");
```

Comment:
Using printf() here in the kernel doesn't appear correct

Reply:
it does, i tested on the on the f411e-disco

Reply:
please polish your code.